src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedAboutToSubmitHandler.java

```diff
@@ -54,14 +56,44 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
-        sscsCaseData.setDwpEditedEvidenceBundleDocument(null);
-        sscsCaseData.setDwpEditedResponseDocument(null);
-        sscsCaseData.setAppendix12Doc(null);
-        sscsCaseData.setDwpUcbEvidenceDocument(null);
+    protected void setDwpDocuments(SscsCaseData sscsCaseData) {
+        if (sscsCaseData.getDwpDocuments() != null) {
+            for (DwpDocument dwpDocument : sscsCaseData.getDwpDocuments()) {
+                if (dwpDocument.getValue().getDocumentType().equals(DwpDocumentType.DWP_RESPONSE.getValue())) {
+                    updateDocument(dwpDocument, sscsCaseData.getDwpResponseDocument(), AppConstants.DWP_DOCUMENT_RESPONSE_FILENAME_PREFIX);
+                    sscsCaseData.setDwpResponseDocument(null);
+                } else if (dwpDocument.getValue().getDocumentType().equals(DwpDocumentType.AT_38.getValue())) {
+                    updateDocument(dwpDocument, sscsCaseData.getDwpAT38Document(), AppConstants.DWP_DOCUMENT_AT38_FILENAME_PREFIX);
+                    sscsCaseData.setDwpAT38Document(null);
+                } else if (dwpDocument.getValue().getDocumentType().equals(DwpDocumentType.DWP_EVIDENCE_BUNDLE.getValue())) {
+                    updateDocument(dwpDocument, sscsCaseData.getDwpEvidenceBundleDocument(), AppConstants.DWP_DOCUMENT_EVIDENCE_FILENAME_PREFIX);
+                    sscsCaseData.setDwpEvidenceBundleDocument(null);
+                }
+            }
+        }
+
+        if (sscsCaseData.getDwpAT38Document() != null || sscsCaseData.getDwpEvidenceBundleDocument() != null || sscsCaseData.getDwpResponseDocument() != null) {
+            String todayDate = java.time.LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy"));
+            dwpDocumentService.moveDocsToCorrectCollection(sscsCaseData, todayDate);
+        }
+    }
+
+    private void updateDocument(DwpDocument dwpDocument, DwpResponseDocument dwpResponseDocument, String documentTypePrefix) {
+        String todayDate = java.time.LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy"));
+        String fileExtension = dwpResponseDocument.getDocumentLink().getDocumentFilename()
+                .substring(dwpResponseDocument.getDocumentLink().getDocumentFilename().lastIndexOf("."));
+        if (dwpResponseDocument != null) {
+            if (!dwpDocument.getValue().getDocumentLink().getDocumentUrl()
+                    .equals(dwpResponseDocument.getDocumentLink().getDocumentUrl())) {
+                dwpDocument.getValue().setDocumentLink(DocumentLink.builder()
+                        .documentBinaryUrl(dwpResponseDocument.getDocumentLink().getDocumentBinaryUrl())
+                        .documentUrl(dwpResponseDocument.getDocumentLink().getDocumentUrl())
+                        .documentFilename(documentTypePrefix + " on " + todayDate + fileExtension)
+                        .build());
+                dwpDocument.getValue().setDocumentDateAdded(java.time.LocalDate.now().toString());
```

Comment:
Isn't `java.time.LocalDate.now().toString()` the same as `java.time.LocalDate.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy"))`?

Reply:
Yes but its not formatted, it might be different based on OS config